app/models/server/raw/LivechatDepartment.ts

```diff
@@ -5,6 +5,10 @@ import { BaseRaw } from './BaseRaw';
 import { ILivechatDepartmentRecord } from '../../../../definition/ILivechatDepartmentRecord';
 
 export class LivechatDepartmentRaw extends BaseRaw<ILivechatDepartmentRecord> {
+	findOneByIdOrName(_department: number): Error {
```

Comment:
Hey! Pls copy this function from `LivechatDepartment.js` on the normal models file 😬